common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -47,6 +48,9 @@
   private static final String SHAPLEY_ENABLE_PROPERTY = "shapley.enable";
   private static final boolean ENABLE_SHAPLEY_CONTRIBUTION =
           Boolean.getBoolean(SHAPLEY_ENABLE_PROPERTY);
+  private static final String SHAPLEY_ENABLED_TYPES_PROPERTY = "shapley.types.enabled";
```

Comment:
Awesome work keeping current env var around for backward-compatibility 👍🏾 👍🏾 !

In case you're looking for a trick to not having to keep `ENABLE_SHAPLEY_CONTRIBUTION` around after initial check, some type of logic that folds this into `ENABLED_SHAPLEY_TYPES` could be:

(psuedocode, using ternary only as example)
```
SHAPLEY_ENABLED_TYPES_PROPERTY  = ENABLE_SHAPLEY_CONTRIBUTION ? "ALL" : howeverShaplyEnabledTypesIsUsuallySet()
```

And only `SHAPLEY_ENABLED_TYPES_PROPERTY` afterward.